app/invocation/invocation_buttons.tsx

```diff
@@ -26,6 +27,9 @@ export default class InvocationButtons extends React.Component<InvocationButtons
         {this.props.model.isWorkflowInvocation() && this.canRerunWorkflow() && (
           <WorkflowRerunButton model={this.props.model} />
         )}
+        {this.props.model.getIsRBEEnabled() && this.props.model.isInProgress() && (
```

Comment:
I'm not sure if this will work for regular Bazel invocations -- we probably want to change the condition here from `getIsRBEEnabled` to `isWorkflowInvocation`

Reply:
@vadimberezniker [This Cancel method](https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/remote_execution/execution_server/execution_server.go#L934) should work for both regular Bazel + workflow invocations right?

Based on the code it looks generalizable but when I try to cancel a Bazel build locally it continues to completion. Having trouble verifying if that's just bc the build is too fast, so almost everything has already been executed by the time I'm able to cancel

Reply:
Are you testing with "remote execution" enabled? The Cancel API cancels all the remote actions associated with an invocation, it does not cancel the invocation itself. So if you are manually running bazel w/o remote execution enabled nothing should happen.

Reply:
Since https://github.com/buildbuddy-io/buildbuddy/pull/1858, I think the "outer" workflow invocation is now associated with the CI runner action, so cancelling the workflow invocation should cancel the CI runner execution and as a result the nested Bazel process should be killed -- which I guess would result in a disconnected inner invocation

Reply:
Done